db/migrate/20250207080511_create_api_credentials.rb

```diff
@@ -0,0 +1,19 @@
+class CreateApiCredentials < ActiveRecord::Migration[7.2]
+  def change
+    create_table :api_credentials do |t|
+      t.references :user, null: false, foreign_key: true
+      t.string :api_token
```

Comment:
you're also storing the plain text tokens in the table?

Reply:
nvm I think you created a migration to remove them right?

Reply:
Yes, originally added unneeded token columns then created a migration to remove them.

Reply:
cool